pho-chat/src/components/providers/revenuecat-provider.test.ts

```
@@ -0,0 +1,42 @@
+import React from "react";
+import { render } from "@testing-library/react";
+
+describe("RevenueCatProvider", () => {
+  const OLD_ENV = process.env;
+  beforeEach(() => {
+    process.env = { ...OLD_ENV, NEXT_PUBLIC_REVENUECAT_API_KEY: "rc_test_key" };
+  });
+
+  it("renders without throwing and calls Purchases.configure", async () => {
+    const { Purchases } = await import("@revenuecat/purchases-js");
+    const el = React.createElement(RevenueCatProvider, { children: React.createElement("div") });
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)